client/layout/sidebar/footer.jsx

```diff
@@ -3,21 +3,49 @@
  */
 import React from 'react';
 import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import page from 'page';
 
 /**
  * Internal dependencies
  */
-var Gridicon = require( 'components/gridicon' );
-
+import Gridicon from 'components/gridicon';
 import Button from 'components/button';
+import { openChat } from 'state/ui/happychat/actions';
+import viewport from 'lib/viewport';
+import config from 'config';
 
-const SidebarFooter = ( { translate, children } ) => (
+const SidebarFooter = ( { translate, children, onOpenChat } ) => (
 	<div className="sidebar__footer">
 		{ children }
 		<Button compact borderless href="/help">
 			<Gridicon icon="help-outline" /> { translate( 'Help' ) }
 		</Button>
+		{ config.isEnabled( 'happychat' )
+			? (
+			<Button className="sidebar__footer__support-chat" compact borderless onClick={ onOpenChat }>
```

Comment:
Also, this class should have only one double underscore (`__`). Everything else should be dashes.